pkg/csi/service/common/constants.go

```diff
@@ -431,6 +431,8 @@ const (
 	WorkloadDomainIsolationFSS = "workload-domain-isolation"
 	// VPCCapabilitySupervisor is a supervisor capability indicating if VPC FSS is enabled
 	VPCCapabilitySupervisor = "VPC_Supported"
+	// TODO: WCP_VMService_BYOK_FSS enables Bring Your Own Key (BYOK) capabilities.
+	BYOK_FSS = "byok"
```

Comment:
Can you confirm if this capability name exists in the required repository on GitLab.

Reply:
@adikul30 We won't use capability but the FSS. 
@divyenpatel has to propagate the WCP_VMService_BYOK_FSS  value into the CSI configmap.